service-loadbalancer/index-template.http

```diff
@@ -0,0 +1 @@
+404 page not found
```

Comment:
> 503 Service Unavailable

Well, 503 is not the "correct code" for a non existing service. If I see 503 it could means that there is no server for a particular backend. 

(If you don't think this is useful I can close this PR)